scale-testing.mk

```diff
@@ -5,13 +5,19 @@
 AWS_REGION ?= us-east-2
 EKS_K8S_VERSION ?= 1.30
 
+VERSION ?= 0.8.1
```

Comment:
Could we document that we don't support testing dev instances yet? We are missing the necessary docker registry e.g. ECR? We will probably want to create the registry so we can dev builds.

Reply:
done here: https://github.com/hashicorp/vault-secrets-operator/pull/916/commits/d4c107427d723ba5440f3905bab07187bc8277dd